dspace-spring-rest/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -514,4 +520,480 @@ public void deleteViolatingConstraints() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        EPerson ePerson = EPersonBuilder.createEPerson(context)
+                                        .withNameInMetadata("John", "Doe")
+                                        .withEmail("Johndoe@fake-email.com")
+                                        .withNetId("testId")
+                                        .build();
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        List<Operation> ops = new ArrayList<Operation>();
+
+        ReplaceOperation replaceOperation1 = new ReplaceOperation("/canLogin", "true");
+        ops.add(replaceOperation1);
+        ReplaceOperation replaceOperation2 = new ReplaceOperation("/netid", "multitestId");
+        ops.add(replaceOperation2);
+        ReplaceOperation replaceOperation3 = new ReplaceOperation("/certificate", "true");
+        ops.add(replaceOperation3);
+        String patchBody = getPatchContent(ops);
+
+        getClient(token).perform(patch("/api/eperson/epersons/" + ePerson.getID())
+                .content(patchBody)
+                .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$.canLogIn", Matchers.is(true)))
+                        .andExpect(jsonPath("$.netid", Matchers.is("multitestId")))
+                        .andExpect(jsonPath("$.requireCertificate", Matchers.is(true)));
+
+    }
+
+    @Test
+    public void patchMultipleOperationsWithFailure() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        EPerson ePerson = EPersonBuilder.createEPerson(context)
+                                        .withNameInMetadata("John", "Doe")
+                                        .withEmail("Johndoe@fake-email.com")
+                                        .build();
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        List<Operation> ops = new ArrayList<Operation>();
+        ReplaceOperation replaceOperation0 = new ReplaceOperation("/canLogin", "true");
+        ops.add(replaceOperation0);
+        String patchBody = getPatchContent(ops);
+
+        // Initialized canLogIn value is true.
+        getClient(token).perform(patch("/api/eperson/epersons/" + ePerson.getID())
+                .content(patchBody)
+                .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$.canLogIn", Matchers.is(true)));
+
+        // The first operation in the series sets canLogIn to be false.
+        ReplaceOperation replaceOperation1 = new ReplaceOperation("/canLogin", "false");
+        ops.add(replaceOperation1);
+        ReplaceOperation replaceOperation2 = new ReplaceOperation("/certificate", "true");
+        ops.add(replaceOperation2);
+        // This will fail.
+        ReplaceOperation replaceOperation3 = new ReplaceOperation("/cert", "false");
```

Comment:
it is not really necessary to use a real boolean here as the path doesn't exist but... 
Maybe it is better to clarify that the operation will fail due to the missing path (or you can use "/unexistentPath", "someValue") to avoid confusion. Someone could assume later that the issue was the textual boolean